### PR TITLE
Ready for cran

### PR DESCRIPTION
**Pull request by jonathandroth**

### CODE REVIEW

File: R/compute_efficient_estimator_and_se.R [hunk 1]
```diff
@@ -1070,41 +1222,110 @@ staggered_cs <- function(df,
...
+  }
+  if(!t %in% colnames_df){
+    stop(paste0("There is no column ", t, " in the data. Thus, we are not able to find the time identifier variable."))
+  }
+  if(!g %in% colnames_df){
+    stop(paste0("There is no column ", g, " in the data. Thus, we are not able to find the group identifier variable."))
+  }
+  if(!y %in% colnames_df){
+    stop(paste0("There is no column ", y, " in the data. Thus, we are not able to find the outcome variable."))
+  }
```
Comment (jonathandroth) [inline R/compute_efficient_estimator_and_se.R:1352]: I think we need to check the user doesn't do something like:

i = "g"
g = "t"

which will mess up the logic below.

Something like


if( i %in% c("g","t","y") ){
stop("Unit identifier cannot be labeled "g", "t" , or "y")
}

if( t %in% c("i","g","y") ){
stop("Time identifier cannot be labeled "i", "g" , or "y")
}

if( g %in% c("i","t","y") ){
stop("Group identifier cannot be labeled "i", "t" , or "y")
}

File: R/compute_efficient_estimator_and_se.R [hunk 2]
```diff
@@ -1070,41 +1222,110 @@ staggered_cs <- function(df,
...
+#' @references
+#'   \cite{Sun, Liyang, and Abraham, Sarah (2020),
+#'   'Estimating dynamic treatment effects in event studies with heterogeneous treatment effects', Forthcoming at the Journal of Econometrics,
+#'   \doi{10.1016/j.jeconom.2020.09.006}.}
+#' @examples
+#' \dontrun{
+#' # Load some libraries
+#' library(dplyr)
+#' library(purrr)
+#' library(MASS)
```
Comment (jonathandroth) [inline R/compute_efficient_estimator_and_se.R:1306]: We should have 

library(staggered)

too right?

File: R/compute_efficient_estimator_and_se.R [hunk 3]
```diff
@@ -1070,41 +1222,110 @@ staggered_cs <- function(df,
...
+#' # load the officer data
+#' df <- pj_officer_level_balanced
```
Comment (jonathandroth) [inline R/compute_efficient_estimator_and_se.R:1308]: I think this should be staggered::pj_officer_level_balanced?

File: R/compute_efficient_estimator_and_se.R [hunk 4]
```diff
@@ -805,23 +819,58 @@ calculate_full_vcv <- function(eventPlotResultsList, resultsDF){
...
+#' # We modify the data so that the time dimension is named t,
+#' # the period of treatment is named g,
+#' # the outcome is named y,
+#' # and the individual identifiers are named i.
+#' df <- df %>% rename(t = period, y = complaints, g = first_trained, i = uid)
```
Comment (jonathandroth) [inline R/compute_efficient_estimator_and_se.R]: Should we do the rename, or do you want to give the examples with your new options (t ="period", y="complaints"...)

Comment (jonathandroth) [reply R/compute_efficient_estimator_and_se.R]: Same comment applied to the staggered_cs and staggered_cs docs

File: R/compute_efficient_estimator_and_se.R (hunk 1 above)
Comment (pedrohcgs) [reply R/compute_efficient_estimator_and_se.R:1352]: agreed!

File: R/compute_efficient_estimator_and_se.R (hunk 2 above)
Comment (pedrohcgs) [reply R/compute_efficient_estimator_and_se.R:1306]: not needed

File: R/compute_efficient_estimator_and_se.R (hunk 3 above)
Comment (pedrohcgs) [reply R/compute_efficient_estimator_and_se.R:1308]: should be data(pj_officer_level_balanced), I think

File: R/compute_efficient_estimator_and_se.R (hunk 4 above)
Comment (pedrohcgs) [reply R/compute_efficient_estimator_and_se.R]: good! I think the new options!

**Pull request merged**